05-UI-ReactJS/src/App.js

```diff
@@ -0,0 +1,74 @@
+import { connect } from 'react-redux'
+import React from 'react';
+import './App.scss';
+import Option from './Option'
+import logo from './logo.svg';
+// import './App.css';
+import MovieForm from './MovieForm';
+import MovieList from './MovieList';
+import { addMovie, removeMovie, toggleMovie, setVisibilityFilter } from './actions';
+
+let App = React.createClass({
+
+  render: function () {
+    return (
+      <div className="App">
+        <div className="header">
+          <img src={logo} className="header--logo" alt="logo" />
+          <h2>Welcome to your movies library</h2>
+        </div>
+        <p className="App--intro">
+          Here you can save all the movies you like!
+        </p>
+        <Option title="Add a new movie" option={<MovieForm movies={this.props.movies} onMovieAdd={this.props.onMovieAdd} />} />
+        <Option title="Watch your list of movies" option={<MovieList filter={this.props.filter} movies={this.props.movies} onMovieRemove={this.props.onMovieRemove}
+        onSetVisibilityFilter={this.props.onSetVisibilityFilter} onToggleMovie={this.props.onToggleMovie} />} />
+      </div>
+    );
+  }
+})
+
+const getVisibleMovies = (movies, filter) => {
+  switch (filter) {
+    case 'SHOW_ALL':
```

Comment:
Seems you are using `SHOW_ALL`, `SHOW_WATCHED` and `SHOW_NON_WATCHED` in plenty of places. You could move it to a constants file.